format/FlightSql.proto

```diff
@@ -867,6 +867,45 @@ enum SqlSupportsConvert {
   SQL_CONVERT_VARCHAR = 19;
 }
 
+/*
+ * Represents a request to retrieve information about data type supported ona Flight SQL enabled backend.
+ * Used in the command member of FlightDescriptor for the following RPC calls:
+ *  - GetSchema: return the schema of the query.
+ *  - GetFlightInfo: execute the catalog metadata request.
+ *
+ * The returned schema will be:
+ * <
+ *   type_name: utf8 not null,
+ *   data_type: int not null,
+ *   column_size: int,
+ *   literal_prefix: utf8,
+ *   literal_suffix: utf8,
+ *   create_params: utf8
+ *   nullable: int not null,
```

Comment:
nit: why is this not a bool?

Reply:
According to the documentation of the JDBC and ODBC this field has three possible values

The possible values are:
```
    typeNoNulls - does not allow NULL values
    typeNullable - allows NULL values
    typeNullableUnknown - nullability unknown 
```